pkg/apis/proto/daemon/daemon.proto

```diff
@@ -114,7 +116,7 @@ service DaemonService {
   };
 
   rpc GetVertexMetrics (GetVertexMetricsRequest) returns (GetVertexMetricsResponse) {
-    option (google.api.http).get = "/api/v1/pipelines/{pipeline}/vertices/{vertex}/metrics";
+    option (google.api.http).get = "/api/v1/pipelines/{pipeline}/vertices/{vertex}/pods/{pods}/metrics";
```

Comment:
Should we introduce a new API, e.g. `GetVertxItemizedMetrics`, which returns metrics for each pod, and is supposed to be only used by reduce vertex? With that, we could get least changes for the backend.

Reply:
can do this